packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -373,10 +388,9 @@ Outputs:
     Description: The name of the ECR repository which hosts docker image assets
     Value:
       Fn::Sub: "${ContainerAssetsRepository}"
+  # The Output is used by the CLI to verify the version of the bootstrap resources.
```

Comment:
Do we really need this to be validated both by CFN and the CLI? Maybe we can omit the CLI now?

Reply:
CLI also does the check before uploading assets and, in the future, before querying context. 

Even if we start reading the SSM parameter today (which is not a bad idea) we'll still need the the CLI to support old bootstrap stacks.